NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/CraftServer.java

```diff
@@ -290,23 +293,47 @@ public void loadPlugins() {
+                                            "In version 4.6.0 and lower, ProtocolLib does not work as expected due to a netty update.\n" +
+                                            "So.. once again, please update!\n" +
+                                            "You can download the latest version with this link: " +
+                                            "https://github.com/dmulloy2/ProtocolLib/releases/latest\n" +
+                                            "Sleeping for 10s so this message can be read."
+                            );
+                            Thread.sleep(10000);
                         }
                     }
                     // Nacho end
                     // Nacho start - [Nacho-0044] Fix Citizens
                     else if(plugin.getDescription().getFullName().contains("Citizens")) {
-                        boolean val = RuntimePatches.applyCitizensPatch(plugin).join();
-                        if(val) {
-                            Logger.getLogger(CraftServer.class.getName()).log(Level.INFO, "Callback returned a good state, Citizens patch was successful and Citizens is now loading.");
-                        } else {
-                            Logger.getLogger(CraftServer.class.getName()).log(Level.SEVERE, "An error occurred trying to patch Citizens, the plugin will not work as expected!");
+                        if(PluginUtils.getCitizensBuild(plugin) < 2396) {
+                            logger.warning(
+                                    "Please update to Citizens 2.0.28 #7 or higher!\n" +
+                                         "In older versions, we have to do a nasty fix which only works in Java versions before Java 11.\n" +
+                                         "So.. once again, please update!\n" +
+                                         "You can download the latest version with this link: " +
+                                         "https://www.spigotmc.org/resources/citizens.13811/updates\n" +
+                                         "This fix may be removed at any moment without notice!\n" +
+                                         "Sleeping for 10s so this message can be read."
+                            );
+                            Thread.sleep(10000);
+                            if(SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_11)) {
+                                logger.warning("Unable to patch Citizens on Java " + System.getProperty("java.specification.version") + ", the plugin will not work as expected!");
+                                Thread.sleep(3000);
+                            } else {
+                                boolean val = RuntimePatches.applyCitizensPatch(plugin).join();
+                                if (val) {
+                                    Logger.getLogger(CraftServer.class.getName()).log(Level.INFO, "Callback returned a good state, Citizens patch was successful and Citizens is now loading.");
```

Comment:
This little piece of text is useless.
```suggestion
                                    Logger.getLogger(CraftServer.class.getName()).log(Level.INFO, "Citizens patch was successful and Citizens is now loading.");
```